Docker/entrypoint.sh

```diff
@@ -12,7 +12,7 @@ if [ -n "$LISTEN" ]; then
 fi
 
 if [ -n "$TRUSTED_PROXY" ]; then
-	if [ "$TRUSTED_PROXY" -eq 0 ]; then
+	if [[ "$TRUSTED_PROXY" == 0 ]]; then
```

Comment:
Please double-check that is works both with our Debian and Alpine images (which uses `ash`).
This line might have to be transformed to a standard POSIX equivalent, that does not require Bash.

Reply:
You're right, I had assumed bash.
I'll fix that

Reply:
This time I've tested with both Dockerfile and Dockerfile-Alpine